internal/core/window.rs

```diff
@@ -279,9 +279,19 @@ impl<'a> WindowProperties<'a> {
     }
 
     /// Returns true if the window should be shown fullscreen; false otherwise.
-    pub fn fullscreen(&self) -> bool {
+    pub fn is_fullscreen(&self) -> bool {
```

Comment:
Unfortunately, we can't rename this function since it is already public in previous version.
We'll have to re-add it as a deprecated function.

Or the alternative is to keep the old name and leave the `is_` prefix out of `WindowProperties`.  But i think it's better to be consistent and keep the `is_` prefix for all the window states

Reply:
Makes sense, I added back `fullscreen()` to both `Window` and `WindowProperties` as deprecated methods